packages/account-v2/src/modules/POI/__tests__/POI.spec.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import { useKycAuthStatus, useResidenceList } from '@deriv/api-v2';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { usePOIInfo } from '../../../hooks';
+import { ProofOfIdentity } from '../POI';
+
+const mockKycAuthStatusResponse = {
+    isLoading: false,
+    kyc_auth_status: {
+        address: {
+            status: 'none',
+        },
+        identity: {
+            service: '',
+            status: 'none',
+        },
+    },
+};
+
+jest.mock('@deriv/api-v2', () => ({
+    useIdentityDocumentVerificationAdd: jest.fn().mockReturnValue({ mutateAsync: jest.fn() }),
+    useKycAuthStatus: jest.fn().mockReturnValue(mockKycAuthStatusResponse),
+    useResidenceList: jest.fn().mockReturnValue({ data: [] }),
+    useSettings: jest.fn().mockReturnValue({ data: {}, isLoading: false, mutation: { mutateAsync: jest.fn() } }),
+}));
+
+jest.mock('../../../hooks', () => ({
+    usePOIInfo: jest.fn().mockReturnValue({ isLoading: false, kycAuthStatus: {} }),
+}));
+
+jest.mock('@deriv-com/ui', () => ({
+    ...jest.requireActual('@deriv-com/ui'),
+    useDevice: jest.fn().mockReturnValue({ isMobile: false }),
+}));
+
+describe('POI', () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it('should render loader', () => {
+        (useKycAuthStatus as jest.Mock).mockImplementation(() => ({
+            ...mockKycAuthStatusResponse,
+            isLoading: true,
+        }));
+        render(<ProofOfIdentity />);
+        expect(screen.getByTestId('dt_derivs-loader')).toBeInTheDocument();
+    });
+
+    it('should render VerificationStatus for IDV when status is verified', async () => {
+        (useKycAuthStatus as jest.Mock).mockReturnValue({
+            isLoading: false,
+            kyc_auth_status: {
+                address: {
+                    status: 'none',
+                },
+                identity: {
+                    service: 'idv',
+                    status: 'verified',
+                },
+            },
+        });
+        render(<ProofOfIdentity />);
+        await waitFor(() => {
+            expect(screen.getByText('ID verification passed')).toBeInTheDocument();
+        });
+    });
```

Comment:
maybe just to simplify:
```suggestion
            expect(await screen.findByText('ID verification passed')).toBeInTheDocument();
    });
```
you can ignore it, or apply in other places as well :)

Reply:
Thanks for assertions I will make this change.